dlairflow/test/test_util.py

```diff
@@ -8,10 +8,18 @@
 from .test_postgresql import temporary_airflow_home
 
 
-def test_user_scratch():
+def test_user_scratch(monkeypatch):
     """Test scratch dir.
     """
-    assert user_scratch() == os.path.join('/data0', 'datalab', os.environ['USER'])
+    monkeypatch.setenv('DLAIRFLOW_SCRATCH_ROOT', '/data0/datalab')
+    assert user_scratch('owner') == os.path.join('/data0', 'datalab', 'owner')
+
+
+def test_user_scratch_exception():
+    """Test scratch dir when environment variable is missing.
+    """
+    with pytest.raises(KeyError):
+        foo = user_scratch('owner')
```

Comment:
What happens if I have `DLAIRFLOW_SCRATCH_ROOT` defined but empty eg. `DLAIRFLOW_SCRATCH_ROOT=`. Perhaps the method should also error in that scenario or would the assumption be that the working directory is used as the root at that point?